owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -970,6 +957,15 @@ private boolean onFileActionChosen(int menuId) {
                 }
                 return true;
             }
+            case R.id.action_send_file: {
```

Comment:
@abelgardep https://github.com/owncloud/android/pull/3638/files/4f8f192987c4f39e2003928e7e1566641d00b7be#r858889730
Now this is here. Before, it was in a switch of actions on only one file. Now, it is in the switch of actions on a batch of files